File: src/main/java/kr/ac/kumoh/k20230626/javawebspring/controller/LottoController.java
package kr.ac.kumoh.k20230626.javawebspring.controller;

import kr.ac.kumoh.k20230626.javawebspring.service.LottoSeervice;
import org.springframework.stereotype.Controller;
import org.springframework.ui.Model;
import org.springframework.web.bind.annotation.GetMapping;

@Controller
public class LottoController {
    private final LottoSeervice lottoSeervice;

    public LottoController(LottoSeervice lottoSeervice) {
        this.lottoSeervice = lottoSeervice;
    }

    @GetMapping("/lotto/numbers")
    public String getLottoNumbers(Model model) {
        model.addAttribute("lotto", lottoSeervice.getLuckNumber().getNumbers());
        return "lotto";
    }
}

File: src/main/kotlin/kr/ac/kumoh/k20230626/lotto/controller/LottoController.kt
package kr.ac.kumoh.k20230626.lotto.controller

import kr.ac.kumoh.k20230626.lotto.Service.LottoService
import org.springframework.stereotype.Controller
import org.springframework.ui.Model
import org.springframework.web.bind.annotation.GetMapping

@Controller
class LottoController(val lottoService: LottoService) {
    @GetMapping("/lotto/numbers")
    fun fetchLottoNumbers(model: Model): String {
        model.addAttribute("lottoNumbers", lottoService.fetchLottoNumbers())
        return "lotto"
    }
}